spec/models/case_attachment_upload_group_spec.rb

```diff
@@ -34,8 +28,22 @@
   end
 
   describe "date_time" do
-    it "returns the formatted upload group time converted to BST" do
-      expect(upload_group.date_time).to eq "08 Jun 2017 11:11"
+    context "when timezone offset is BST" do
```

Comment:
Looking at this now has made me realise there is no point testing GMT and BST because nothing is being done with the time zone anymore. 
So a test that checks the correct format is returned is all that is required.